controllers/TodoController.js

```diff
@@ -0,0 +1,118 @@
+const {Todo} = require('../models')
+
+class TodoController {
+    static async create(req, res) {
+        const query = {
```

Comment:
Karena disini kita ingin menambahkan todo baru, maka better nama object nya menjadi `todo`

```suggestion
        const todo = {
```